service/controller/v4/cloudconfig/template.go

```diff
@@ -4,7 +4,10 @@ const (
 	calicoAzureFileName       = "/srv/calico-azure.yaml"
 	calicoAzureFileOwner      = "root:root"
 	calicoAzureFilePermission = 0600
-	calicoAzureFileTemplate   = `# Calico Version v3.2.0
+	calicoAzureFileTemplate   = `# Extra changes:
+#  - added "nodename_file_optional" set to true (can be removed on the next upgrade)
```

Comment:
Lets please have a TODO issue in the operator board with more information about the versions and reasons and please use appropriate labels. <3

Reply:
added https://github.com/giantswarm/giantswarm/issues/4113